src/components/form-helpers/form-helpers.js

```diff
@@ -320,9 +321,17 @@ const _PhoneFieldItem = ({
       autoPlaceholder: 'aggressive',
       preferredCountries: ['ar', 'mx', 'co', 'es', 'ec', 'cl', 'pe', 'us'],
       initialCountry: 'auto',
-      geoIpLookup: (callback) => {
-        // TODO: determine current country using geolocation
-        callback('ar');
+      geoIpLookup: async (success, failure) => {
+        // TODO: read cache before request
+        try {
+          const response = await axios.get('https://ipinfo.io/json');
+          var countryCode = response && response.country ? response.country : '';
```

Comment:
Oh, it seems that our CI process does not take care of this :disappointed: 

We should have to have special attention to this (and fix our CI).

```suggestion
          const countryCode = response && response.country ? response.country : '';
```